plugins/deploy_waitlist/app/controllers/deploy_waitlist_controller.rb

```diff
@@ -0,0 +1,42 @@
+class DeployWaitlistController < ApplicationController
+
+  def show
+    respond_to do |format|
+      format.html
+      format.json { render json: current_waitlist.to_json }
+    end
+  end
+
+  def add
+    Rails.logger.warn("current_waitlist: #{current_waitlist.list}")
```

Comment:
this is good for our debugging, but in a PR to zendesk, i would advise making this debug